### DISCUSSION TITLE
posts/on-system-security-research/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#9] [Announcements]
# posts/on-system-security-research/

系统安全研究：借鉴与思考

https://blog.wohin.me/posts/on-system-security-research/

**Comment by CatherinePeng2023** [comment]
信息量很大！特别是研究思维的范式总结，以及总结出的个人研究流程范式。不过“造锤找钉”这个研究范式要么是颠覆性的，要么出成果遥遥无期，得慎重，牛啊牛啊！

**Reply by brant-ruan** [reply]
👻👻好有道理！

**Comment by ListenerMoya** [comment]
定期来逛逛，都有新收获~

**Reply by brant-ruan** [reply]
你也开一个blog 🤔

**Comment by Ainevsia** [comment]
感谢大佬分享

**Comment by AmiaaaZ** [comment]
感谢分享！
